infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/utils/LogReplicationConfigManager.java

```diff
@@ -50,7 +47,6 @@
 @Slf4j
 public class LogReplicationConfigManager {
```

Comment:
Guess this needs rebasing now
The majority of change in this file is in https://github.com/CorfuDB/CorfuDB/pull/3302/files#diff-178cf06dab1224a243aa8ae8dda938910fda09d5a3c4f44f11db95c47b61b90a

Reply:
Rebased, thx!